src/main/java/ru/practicum/shareit/item/dto/ItemDto.java

```diff
@@ -1,7 +1,14 @@
 package ru.practicum.shareit.item.dto;
 
-/**
- * TODO Sprint add-controllers.
- */
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
 public class ItemDto {
+    private int id;
+    private String name;
+    private String description;
+    private Boolean available = false;
```

Comment:
Инициализация на данном уровне лишняя
```suggestion
    private Boolean available;
```

Reply:
+